Error in running case 'Shape_detection'

_Please use the following template to help us solving your issue._

## Issue Details

Error in running case 'Shape_detection' 

```
1>efficient_RANSAC_with_point_access.cpp
1>E:\code\CGAL-5.6\include\CGAL/Shape_detection/Efficient_RANSAC/Octree.h(46,1): error C2947: expecting '>' to terminate template-argument-list, found '<'
1>E:\code\CGAL-5.6\include\CGAL/Shape_detection/Efficient_RANSAC/Octree.h(46,66): error C2365: 'Gt': redefinition; previous definition was 'template parameter'
1>E:\code\CGAL-5.6\include\CGAL/Shape_detection/Efficient_RANSAC/Octree.h(45): message : see declaration of 'Gt'
1>E:\code\CGAL-5.6\include\CGAL/Shape_detection/Efficient_RANSAC/Octree.h(46,68): error C2988: unrecognizable template declaration/definition
1>E:\code\CGAL-5.6\include\CGAL/Shape_detection/Efficient_RANSAC/Octree.h(46,68): error C2059: syntax error: ','
1>E:\code\CGAL-5.6\include\CGAL/Shape_detection/Efficient_RANSAC/Octree.h(47,1): error C2143: syntax error: missing ';' before '{'
1>E:\code\CGAL-5.6\include\CGAL/Shape_detection/Efficient_RANSAC/Octree.h(47,1): error C2447: '{': missing function header (old-style formal list?)
1>Done building project "efficient_RANSAC_with_point_access.vcxproj" -- FAILED.
```

## Source Code

 

## Environment

* Operating system (Windows/Mac/Linux, 32/64 bits):windows 10 64bits
* Compiler:Visual Studio 2019
* Release or debug mode:debug
* Specific flags used (if any):
* CGAL version:5.6
* Boost version:1.83.0
* Other libraries versions if used (Eigen, TBB, etc.):

--- a/Shape_detection/include/CGAL/Shape_detection/Efficient_RANSAC/Octree.h
+++ b/Shape_detection/include/CGAL/Shape_detection/Efficient_RANSAC/Octree.h
@@ -31,20 +31,12 @@
 namespace CGAL {
 namespace Shape_detection {
 
-// Forward declaration needed for automatic traits detection without
-// including the deprecated header itself…
-template <typename Gt, typename IR, typename IPM, typename INM>
-struct Shape_detection_traits;
-
 namespace internal {
 
 template <typename Traits>
 struct Traits_base { typedef Traits type; };
 template <typename Gt, typename IR, typename IPM, typename INM>
 struct Traits_base<CGAL::Shape_detection::Efficient_RANSAC_traits<Gt,IR,IPM,INM> >
-{ typedef Gt type; };
-template <typename Gt, typename IR, typename IPM, typename INM>
-struct Traits_base<CGAL::Shape_detection::Shape_detection_traits<Gt,IR,IPM,INM> >
 { typedef Gt type; };
 
 template<class Traits>

